### PR TITLE
[Editor] Add a label to the highight floating button readable with NVDA (bug 1886964)

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

**Review by Snuffleupagus — APPROVED** [review]

File: l10n/en-US/viewer.ftl [hunk 1]
```diff
@@ -320,6 +320,8 @@ pdfjs-editor-highlight-button =
 pdfjs-editor-highlight-button-label = Highlight
 pdfjs-highlight-floating-button =
     .title = Highlight
+    .aria-label = Highlight
```
Comment (flodolo) [inline l10n/en-US/viewer.ftl:323]: You need a new string ID to add an attribute
https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html

**Review by flodolo — CHANGES_REQUESTED** [review]

**Review by flodolo — APPROVED** [review]

File: l10n/en-US/viewer.ftl (hunk 1 above)
Comment (Snuffleupagus) [reply l10n/en-US/viewer.ftl:323]: > You need a new string ID to add an attribute
> https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html

Could the Fluent linter be configured to catch that?

Comment (flodolo) [reply l10n/en-US/viewer.ftl:323]: Technically, that's available in compare-locales. I've never tried to integrate it in a workflow though
https://github.com/mozilla/compare-locales/blob/main/compare_locales/lint/cli.py#L21-L24

**Pull request merged**